backend/src/services/database.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+import mongoose from 'mongoose';
+
+export const initDatabase = (MONGO_URL: string) => {
+  mongoose
+    .connect(MONGO_URL)
+    .then(() => console.log('Successfully connected to DB'))
+    .catch((e) => {
+      console.log('Failed to connect to DB ', e);
+      setTimeout(() => {
+        console.log(e);
+      }, 5000);
```

Comment:
Is there specific reason of waiting 5 seconds before writing error to console?

Reply:
Good catch! I actually just moved the entire function from `index.ts`.

I will remove this additional logging statement. Thanks for spotting!

Reply:
Don't work too much on Logging. I will be adding Central Logging as well using `winston` and `Loki` by Grafana.